gcloud/dataflow.yaml

```
@@ -0,0 +1,261 @@
+            overriden using `.ctx.subnetwork`
+        - name: project
+          description: >
+            default project used to access the dataflow API if
+            :code:`.ctx.project` is not provided, optional
+
+      notes:
+        - The system can share data with a common configuration Google Cloud system that
+          contains the configuration.
+        - For example
+        - example: |
+            ---
+            systems:
+              dataflow:
+                extends:
+                  - gcloud-config
+              gcloud-config:
+                project: my-gcp-project
+                locations:
+                  default: us-central1
+                subnetworks:
+                  default: default
+                service_accounts:
+                  dataflow: ENC[gcloud-kms,xxxxxxx]
+
+    data:
+      joblinks:
+        dataflow_job:
+          url: "https://console.cloud.google.com/dataflow/jobs/{{ .sysData.locations.backup }}/{{ .data.job.id }}?project={{ .sysData.project }}"
+          text: See Dataflow job {{ .data.job.name }} in console
+
+    functions:
+      createJob:
+        driver: gcloud-dataflow
+        rawAction: createJob
+        parameters:
+          service_account: $?sysData.service_accounts.dataflow
+          project: $ctx.project,sysData.project
+          location: $ctx.location,sysData.locations.dataflow,sysData.locations.default
+          job: >-
+            :yaml:{{ merge (
+            dict "subnetwork" (coalesce .ctx.subnetwork .sysData.subnetworks.dataflow .sysData.subnetworks.default)
+            | dict "environment"
+            ) .ctx.job | toJson }}
+        export_on_success:
+          job: $data.job
+          links: $sysData.joblinks
+
+        description: >
+          Creates a dataflow job using a template.
+
+        meta:
+          inputs:
+            - name: project
+              description: Optional, in which project the job is created, defaults to the :code:`project` defined with the system
+            - name: location
+              description: Optional, the location for the job, defaults to the system configuration
+            - name: subnetwork
+              description: Optional, the subnetwork for the job, defaults to the system configuration
+            - name: job
+              description: Required, the data structure describe the :code:`CreateJobFromTemplateRequest`, see the `API document <https://pkg.go.dev/google.golang.org/api/dataflow/v1b3#CreateJobFromTemplateRequest>`_ for details.
+          exports:
+            - name: job
+              description: The job object, details `here <https://pkg.go.dev/google.golang.org/api/dataflow/v1b3#Job>`_
+
+          notes:
+            - For example
+            - example: |
+                call_function: dataflow.createJob
+                with:
+                  job:
+                    gcsPath: gs://dataflow-templates//Cloud_Spanner_to_GCS_Avro
```

minor, possible typo?
```suggestion
                    gcsPath: gs://dataflow-templates/Cloud_Spanner_to_GCS_Avro
```